server/huntflow_reloaded/handler.py

```diff
@@ -159,34 +169,44 @@ def _process_request(self):
 
         self._logger.info("Handling 'status' request")
 
-        event = self._decoded_body['event']
-        if not event.get('calendar_event'):
+        self.event = self._decoded_body['event'] # pylint: disable=attribute-defined-outside-init
+
+        self._get_basic_attrs()
+
+        if self.event.get('calendar_event'):
+            await self.handle_calendar_event()
+        elif self.event.get('employment_date'):
+            await self.handle_employment_date()
+        else:
             raise IncompleteRequest
 
+        self._scheduler.publish_now(self.message)
+        await self._scheduler.create_event(self.event_type, context=self.context)
+
+    async def handle_calendar_event(self):
```

Comment:
Why are some of your methods are starts with underline and some of them without it?